CONTRIBUTING.rst

```diff
@@ -925,6 +925,65 @@ returned by the ``request_mocker`` pytest fixture, defined in
 ``Sender`` class it contains provide information on how to write a test using
 this fixture. Existing tests can also serve as examples.
 
+Performance monitoring
+----------------------
+
+As of March 2025, we are starting to incorporate performance monitoring in the
+library using `asv <https://asv.readthedocs.io/en/latest/index.html>`_
+benchmarks. The goal is two-fold:
+
+- To track the performance over time and flag regressions due to changes in the
+  codebase.
+- To compare the performance of different implementations of an algorithm
+  (for example, loading an image using :func:`nilearn.image.load_img` vs.
+  :func:`nibabel.loadsave.load`).
+
+A collection of these benchmarks are located in the ``asv_benchmarks``
+directory. Currently, we run them on GitHub CI everyday on the latest commit
+of the main branch. The results are available on
+`nilearn.github.io/benchmarks/ <https://nilearn.github.io/benchmarks/>`_.
+
+To run these benchmarks locally, you will need to install
+the asv package:
+
+.. code-block:: bash
+
+      pip install asv
+
+Then, change to the ``asv_benchmarks`` directory:
+
+.. code-block:: bash
+
+      cd asv_benchmarks
+
+To run a specific benchmark on the current HEAD of your clone of the
+repository, use the following command:
```

Comment:
Maybe explain where we find all the available benchmarks.

Reply:
I mentioned that in L941. Could also add link